benches/filter.rs

```diff
@@ -83,7 +83,7 @@ pub fn criterion_benchmark(c: &mut Criterion) {
             group.bench_function("Rust/Interleaved", |b| {
                 b.iter(|| {
                     f.process(
-                        black_box(&ebur128::Interleaved::new(&*data, 2).unwrap()),
+                        black_box(ebur128::Interleaved::new(&*data, 2).unwrap()),
```

Comment:
Can you move all the reference changes to a separate commit? That makes the commit history a bit easier to read in the future :)